README.md

```diff
@@ -108,7 +111,10 @@ Shipwright supports any tool that can build container images in Kubernetes clust
 * Wait until your buildrun is completed:
 
   ```bash
-  $ kubectl get buildruns
```

Comment:
And same idea here, we should leave the `$` sign to shown it's an command output.

Reply:
same question as above comment https://github.com/shipwright-io/build/pull/794#discussion_r646655160